ukrainian/JDstory/JDstory.md

```diff
@@ -0,0 +1,2 @@
+�ЫІсторія просто�го JD розробника :)
```

Comment:
It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.